### DISCUSSION TITLE
[PSD7] Exercise 2.26

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#154] [Esercizi (A-L)]
![image](https://user-images.githubusercontent.com/69389461/210018787-0d6b9e65-2741-42fa-8f4b-3c34c6bad028.png)

**Comment by Elia-Belli** [comment]
![BP1](https://user-images.githubusercontent.com/107944476/211665175-309baa70-7d25-4948-aa59-a3bd1c1784ea.jpg)
$Y = (\bar A + \bar B)(\bar C + \bar D) + \bar E$

**Comment by fraxb04** [comment]
corretto
